modules/adnuntiusBidAdapter.js

```diff
@@ -68,10 +68,10 @@ export const spec = {
       } else return response
     }, {});
 
-    const bidResponse = bidRequest.bid.map(bid => bid.bidId)
-      .reduce((request, adunitId) =>
-        request.concat(bidResponsesById[adunitId])
-        , []);
+    const bidResponse = bidRequest.bid.map(bid => bid.bidId).reduce((request, adunitId) => {
+      if (bidResponsesById[adunitId]) request.push(bidResponsesById[adunitId])
```

Comment:
The request.push doesn't appear to be scoped to the if statment block. Am I looking at this wrong?

Reply:
you are correct, I look at the bidResponsesById object if it contains the adunitId, and if it does I will push that object into the request array that will be returned to the auction as winners. this is just to make sure that the order of adunits will be correct.

Reply:
I can make it a bit more clear what happens in the code if you wish?

Reply:
I've honestly never seen an if statment in javascript without brackets so freaks me out. lol

Reply:
Haha i can add brackets to it if you wish. :)

Reply:
And no, nothing will change.

Reply:
thanks 🙏

Reply:
We're in business now. :P ( = should be fixed now)